ci-operator/step-registry/ipi/install/vsphere/registry/ipi-install-vsphere-registry-commands.sh

```diff
@@ -22,7 +22,7 @@ spec:
       storage: 10Gi
 EOF
   oc create -f "${SHARED_DIR}/registry-pvc.yaml" -n openshift-image-registry &&
-  oc patch config.imageregistry.operator.openshift.io/cluster --type=merge -p '{"spec":{"managementState":"Managed","rolloutStrategy":"Recreate","replicas":1,"storage":{"managementState":"Managed","pvc":{"claim":"registry-pvc"}}}}'
+  oc patch config.imageregistry.operator.openshift.io/cluster --type=merge -p '{"spec":{"managementState":"Managed","rolloutStrategy":"RollingUpdate","replicas":2,"storage":{"managementState":"Managed","pvc":{"claim":"registry-pvc"}}}}'
```

Comment:
What is the purpose of updating the `replicas` to `2` ?

Reply:
@jianlinliu  @flavianmissi thought maybe it's safer to update replicas to 2,  @flavianmissi could you help to confirm it?

Reply:
@jianlinliu  @flavianmissi  seems if update rolloutStrategy to RollingUpdate, failed to install vsphere  cluster, so i revert it and replica set to 1, now all jobs succeed, please review again, thanks

Reply:
/lgtm

Reply:
/lgtm

Reply:
the idea behind changes the replicas to 2 was to make the registry highly available. it seems we don't have a choice but to proceed with one replica though - in that case let's try with only the storage change and see where that takes us.

Reply:
okay, seem like we are ready to move on now.

Reply:
@wewang58 did we get issues when the number of replicas was increased to 2, or was the job failure related to the rollout strategy change to RollingUpdate?
I'm asking because I think we might need to give the replica increase a shot - from looking at the [graphs](https://grafana-loki.ci.openshift.org/d/ISnBj4LVk/disruption?orgId=1&var-platform=metal&var-percentile=P50&var-backend=image-registry-new-connections&var-backend=image-registry-reused-connections&var-backend=ingress-to-console-new-connections&var-backend=ingress-to-console-reused-connections&var-releases=4.15&var-upgrade_type=minor&var-upgrade_type=micro&var-networks=sdn&var-networks=ovn&var-topologies=ha&var-architectures=amd64&var-min_job_runs=10&var-lookback=7&var-master_node_updated=Y&var-min_disruption_regression=5&var-min_disruption_job_list=0&var-master_nodes_updated=&var-master_nodes_updated=N&var-master_nodes_updated=Y) it's a bit unclear to me whether this made as much difference as we hoped without changing the number of replicas.